presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -1115,6 +1115,60 @@ public void testCountDistinctPerformanceWarning()
         assertTrue(warning.getMessage().contains("COUNT(DISTINCT xxx)"));
     }
 
+    @Test
+    public void testApproxDistinctPerformanceWarning()
+    {
+        WarningCollector warningCollector = analyzeWithWarnings("SELECT approx_distinct(a) FROM t1 GROUP BY b");
```

Comment:
let's have a test to cover two functions in one query and assert we can get two warnings

Reply:
is there a way to add multiple warnings of the same type to warningCollector? it seems that WarningCollector stores warnings in a Map<WarningCode, PrestoWarning> so only one PERFORMANCE_WARNING would get added

Reply:
lol, shall we change it to a multi map?

Reply:
alright, I can work on that